src/prisma/schema.prisma

```diff
@@ -1826,3 +1827,12 @@ model BlacklistedUser {
   emails          String[]
   createdAt       DateTime @default(now())
 }
+
+model AdditionalBlockQuota {
+  id         String   @id @default(uuid()) @db.Uuid
+  spaceId    String   @db.Uuid
+  space      Space    @relation(fields: [spaceId], references: [id], onDelete: Cascade)
+  expiration DateTime
```

Comment:
this can probably just be optional

Reply:
Good point, since it might never have an expiration date. Updated.